source/includes/pipeline_config/_028-origin.md.erb

```diff
@@ -0,0 +1,70 @@
+<%
+local_origin_json = {
+  origin: {
+    type: 'local',
+    file: 'cruise-config.xml'
+  },
+}
+%>
+
+<%=
+describe_sub_object 'The local origin object', json: local_origin_json, html_id: 'pipeline-config-config-xml-origin-object' do
+  type   'String', 'The type of origin.'
+  file   'String', 'The name of config file.'
+end
+%>
+
+<%
+remote_origin_json = {
+  type: 'local',
+  repo: {
+    type: 'git',
+    attributes: {
+      url: 'https://github.com/config-repo/repo',
+      destination: nil,
+      filter: nil,
+      invert_filter: false,
+      name: nil,
+      auto_update: true,
+      branch: 'master',
+      submodule_folder: nil,
+      shallow_clone: false
+    },
+    revision: '0771169c7178c9850ab098b40cc17b24f3d3c7a9'
+  },
+}
+%>
+
+<%=
+describe_sub_object 'The config repo origin object', json: remote_origin_json, html_id: 'pipeline-config-config-repo-origin-object' do
+  type   'String', 'The type of origin.'
+  repo   'Object', 'The [config repository material](#pipeline-config-config-repository-material-object) object'
+end
+%>
+
+
+<%
+material_json = {
+  type: 'git',
+  attributes: {
+    url: 'https://github.com/config-repo/repo',
+    destination: nil,
+    filter: nil,
+    invert_filter: false,
+    name: nil,
+    auto_update: true,
+    branch: 'master',
+    submodule_folder: nil,
+    shallow_clone: false
+  },
+  revision: '0771169c7178c9850ab098b40cc17b24f3d3c7a9'
+}
+%>
+
+<%=
+describe_sub_object 'The config repository material object', json: material_json, html_id: 'pipeline-config-material-object' do
```

Comment:
`remote_origin_json` also contains the `type` attribute, which is not part of config repository material.